doc/modules/mod_mam.md

```diff
@@ -147,7 +148,7 @@ You can change the default settings using extra parameters:
 {mod_mam_meta, [
         {backend, odbc},
 
-        {add_archived_element, true},
+        {no_stanzaid_element, true},
```

Comment:
I changed it as `add_archived_element` set to true is now considered 'not recommended'. Therefore it is not a good idea to put it in an example.